extmod/modbluetooth.c

```diff
@@ -363,6 +371,8 @@ STATIC mp_obj_t bluetooth_ble_irq(size_t n_args, const mp_obj_t *pos_args, mp_ma
         mp_raise_ValueError("invalid callback");
     }
 
+    bluetooth_handle_uninitialized();
```

Comment:
I don't think this is needed here, this function should work ok without the device enabled.

Reply:
Right. Removed.